test/fixtures/basic-import/expected.js

```diff
@@ -1,5 +1,7 @@
+function _interopRequireWildcard(obj) { if (obj && obj.__esModule) { return obj; } else { var newObj = {}; if (obj != null) { for (var key in obj) { if (Object.prototype.hasOwnProperty.call(obj, key)) newObj[key] = obj[key]; } } newObj.default = obj; return newObj; } }
```

Comment:
ah ok, so if we're using `external-babel-helpers`, then this helper wouldn't appear?

In that case, could we add some parallel tests that use that option, so we can ensure that the helpers appear or not following that setting?